circle.yml

```diff
@@ -495,131 +495,99 @@ workflows:
   build-base-images:
     jobs:
       - build-base-image:
-          name: "base 10.22.0"
```

Comment:
name is not a parameter on the build-base-image job. This value is not used.

Reply:
@emilyrohrbough it looks like the name is used by CircleCI to more easily identify the jobs. 

Before this change:

![Screen Shot 2022-01-04 at 7 51 12 PM](https://user-images.githubusercontent.com/2184114/148148023-12bd8803-1bfc-4427-8edf-a55175568f76.png)

After:

![Screen Shot 2022-01-04 at 7 52 54 PM](https://user-images.githubusercontent.com/2184114/148148239-d54b1a5b-f666-4943-9c99-8b9167f5d41e.png)

Reply:
🤦🏻‍♀️ I actually knew that...just forgot I guess/didn't think this through. Thanks for pointing this out.! [`8287b8d` (#568)](https://github.com/cypress-io/cypress-docker-images/pull/568/commits/8287b8d723af371e15df13cf8e21ac197d280c50)